addons/portal/controllers/portal.py

```diff
@@ -91,9 +91,18 @@ def get_records_pager(ids, current):
     if current.id in ids and (hasattr(current, 'website_url') or hasattr(current, 'access_url')):
         attr_name = 'access_url' if hasattr(current, 'access_url') else 'website_url'
         idx = ids.index(current.id)
+        prev_record = idx != 0 and current.browse(ids[idx - 1])
+        next_record = idx < len(ids) - 1 and current.browse(ids[idx + 1])
+
         return {
-            'prev_record': idx != 0 and getattr(current.browse(ids[idx - 1]), attr_name),
-            'next_record': idx < len(ids) - 1 and getattr(current.browse(ids[idx + 1]), attr_name),
+            'prev_record':
+                '%s?access_token=%s' % (getattr(prev_record, attr_name), prev_record._portal_ensure_token())
```

Comment:
Will break if record has `website_url` without inheriting from `portal.mixin`.
It was supported before (for subscription probably, but without certainty), so let's keep it working for the moment.

Reply:
Fixed! Thanks :grin: